src/contrib/cluster/Akka.Cluster.Tools/Singleton/ClusterSingletonManager.cs

```diff
@@ -630,8 +630,8 @@ private void GetNextOldestChange()
 
         private State<ClusterSingletonState, IClusterSingletonData> GoToOldest()
         {
-            Log.Info("Singleton manager [{0}] starting singleton actor", _cluster.SelfAddress);
             var singleton = Context.Watch(Context.ActorOf(_singletonProps, _settings.SingletonName));
+            Log.Info("Singleton manager starting singleton actor [{0}] ", singleton.Path);
```

Comment:
It's like this in akka. Cannot move it upward, since the path is not yet known there. So will change it to started